src/plugins/opensearch_dashboards_react/public/code_editor/editor_theme.ts

```diff
@@ -84,20 +83,20 @@ export function createTheme(
       { token: 'string.key.json', foreground: standardizeColor(euiTheme.euiColorDanger) },
       { token: 'string.value.json', foreground: standardizeColor(euiTheme.euiColorPrimary) },
 
-      { token: 'attribute.name', foreground: standardizeColor(euiTheme.euiColorDanger) },
+      // { token: 'attribute', foreground: standardizeColor(euiTheme.euiCodeBlockAttributeColor) },
```

Comment:
are we sure we ant to delete this https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/DEVELOPER_GUIDE.md#do-not-comment-out-code